pkgs/tools/misc/legdur/default.nix

```
@@ -0,0 +1,24 @@
+{ lib
+, fetchCrate
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "legdur";
+  version = "0.3.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-IKwEL5TKDDMX5AunRZI04MNAuv/w1pD39in8+j41P4Q=";
+  };
+
+  cargoSha256 = "sha256-JLiAwgghgaoINEo10My2Tc6vG9iaNL1mOe4fG/udcGw=";
+
+  meta = with lib; {
+    description = "keep your legacy durable, detect changes to your directories over time";
+    homepage = "https://git.sr.ht/~cyplo/legdur";
```

This link leads to 404 otherwise.
```suggestion
    homepage = "https://hg.sr.ht/~cyplo/legdur";
```